sdk/cosmos/azure-cosmos/azure/cosmos/_retry_utility.py

```diff
@@ -32,8 +32,10 @@
 from . import _resource_throttle_retry_policy
 from . import _default_retry_policy
 from . import _session_retry_policy
+from . import _gone_retry_policy
```

Comment:
question: usually the import statements should be sorted in java & SDK, do we need to sort in Python as well?

Reply:
yes but this is something that I need to do probably in every file, will take care of all of it in one PR later